airflow/providers/apache/hive/plugins/hive_plugin.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
lets maybe on the way change the name of the files to `macros.py` ?
There is no more reason to state `hive_` since now it's in hive provider

Reply:
We removed aws_, azure_, google_ prefix from all files

Reply:
Should we also remove them from "airbyte", "arango",  "...." (just checked the first few)?

This is for providers that have multiple modules in their packages. 

Most of the providers of ours have just a single module - and all classes are in this module:

```
airbyte.hooks.airbyte
airbyte.operators.airbyte
airbyte.sensors.airbyte

arangodb.hooks.arangodb
arangodb.operators.arangodb
arangodb.sensors.arangodb

....

tableau.hooks.tableau
tableau.operators.tableau
tableau.sensors.tableau
```

Should we change those too? 

Are we calling for a new AIP?

Reply:
You are refering to something else.
The `provider_` was removed from everywhere.
In peoviders with single service we do have
tablue.py, hive.py but we dont have tableau_hooks.py

So hive_plugins.py is not right.
If you want hive.py i'm OK.
The reason I suggested macros.py is beacuse plugins can be many things so telling that the file contains macros in the name is more self explained and will avoid adding there unrelated stuff.

Reply:
Yes I had for a while `hive_plugins.py` but it was gone already when you commented (pushed it a moment later).
And we already have `hive.py` in `macros` package (look closer at the current structure).  

<img width="503" alt="Screenshot 2022-12-23 at 09 08 39" src="https://user-images.githubusercontent.com/595491/209298057-8929edcf-0b00-40d4-aed2-15b9e0a83db1.png">


This is a much more generic structure which is future-proof and usable by other provifers:

* plugins/hive.py -> where the plugins live (and they expose macros)
* macros/hive.py  -> where macros live (those referred by the plugins)

If we have listeners - we willl add them to `listeners/hive.py`

Reply:
Looks good